Update README.md (#12)
Financial-Times/github-apps-config-next: README.md

[![CircleCI](https://circleci.com/gh/Financial-Times/github-apps-config-next.svg?style=svg)](https://circleci.com/gh/Financial-Times/github-apps-config-next)

## What's this?
This repository is where [FT Customer Products](https://biz-ops.in.ft.com/Group/customerproducts) keeps our shared configuration files for our GitHub Apps. 

> > We use GitHub Apps to [help manage our hundreds of GitHub repositories](https://github.com/Financial-Times/next/wiki/How-We-Manage-Our-GitHub-Repositories).

Author: Not sure what to do about that. The links are useful for anyone who has access. If they don't have access, why are they reading this README? 

I guess I could add a note to the README to warn the reader that links are private, but that feels odd ...

> > We use GitHub Apps to [help manage our hundreds of GitHub repositories](https://github.com/Financial-Times/next/wiki/How-We-Manage-Our-GitHub-Repositories).

Reviewer: > I guess I could add a note to the README to warn the reader that links are private, but that feels odd ...

I think that's a great suggestion, I've seen the pattern elsewhere. The non-public information could be in a blockquote addressed to FT engineers.

> 📰 **FT People:** This repository is where [FT Customer Products](https://biz-ops.in.ft.com/Group/customerproducts) keeps our shared configuration files for our GitHub Apps.

> * Configuration for each GitHub App: [The `.github/` folder](https://github.com/Financial-Times/github-apps-config-next#the-github-folder)

Author: How about this?
``` diff
+ * The configuration files for each of our GitHub Apps can be found under [the `.github/` folder](https://github.com/Financial-Times/github-apps-config-next#the-github-folder).
```

> * Configuration for managing GitHub Apps: [The `manage-github-apps/` folder](https://github.com/Financial-Times/github-apps-config-next#the-manage-github-apps-folder)

Author: I prefer the original version here, because the context is about managing our apps vs. managing the configuration for each app. It's not obvious for people who don't know what the `manage-github-apps` is. 

How's this?

``` diff
- * Configuration for managing GitHub Apps: [The `manage-github-apps/` folder](https://github.com/Financial-Times/github-apps-config-next#the-manage-github-apps-folder)
+ * The configuration files for managing our GitHub Apps are in [the `manage-github-apps/` folder](https://github.com/Financial-Times/github-apps-config-next#the-manage-github-apps-folder).
```

### The [`.github/`](https://github.com/Financial-Times/github-apps-config-next/tree/master/.github) folder
This folder contains configuration files for different [GitHub Apps](https://developer.github.com/apps/).

#### [`settings.yml`](https://github.com/Financial-Times/github-apps-config-next/blob/master/.github/settings.yml)

> This configures the settings for your GitHub repository. For example, we disable the repository wiki setting, because we already have [a central wiki](https://github.com/Financial-Times/next/wiki). These settings are applied by the ["Settings" GitHub App](https://probot.github.io/apps/settings).

Author: > I would avoid specifying in the README.md any details about what we have configured, as it'll quickly go out of date.

Getting mixed signals here. [The issue](https://github.com/Financial-Times/github-apps-config-next/issues/3) says: "In [the README](/Financial-Times/github-apps-config-next/blob/master/README.md), explain what we use each GitHub App for."

> This configures the settings for your GitHub repository. For example, we disable the repository wiki setting, because we already have [a central wiki](https://github.com/Financial-Times/next/wiki). These settings are applied by the ["Settings" GitHub App](https://probot.github.io/apps/settings).

Reviewer: Yeah, I think it's helpful to detail what GitHub Apps we have configuration for, and for each one, what the name of their configuration file is. But no need for the details on how they have been configured.

  has_projects: true
```

#### [`stale.yml`](https://github.com/Financial-Times/github-apps-config-next/blob/master/.github/stale.yml)  

> These settings are for the ["Stale" GitHub App](https://github.com/probot/stale). It closes abandoned issues and pull requests after a period of inactivity. 🚀*Tip:* You can see all of our open GitHub issues and pull requests in [the Houston.ft.com repositories page](https://houston.ft.com/repositories).

Reviewer: I'm not totally happy with Houston being plastered everywhere yet, given you are the only contributor.

A GitHub search I feel would be more appropriate.

Author: Not exactly plastered everywhere, but that's fine

 
### The [`manage-github-apps/`](https://github.com/Financial-Times/github-apps-config-next/tree/master/manage-github-apps) folder

Each configuration file in this folder contains a list of one or more GitHub App *installations*. We've installed the [Settings](https://probot.github.io/apps/settings), [Stale](https://github.com/probot/stale),  [Renovate](https://github.com/apps/renovate) and [Tako](https://github.com/Financial-Times/tako/blob/master/README.md) Apps in the Financial Times organisation in GitHub. Each of these has an *installation ID*. 

> *Why?* Unfortunately, we can't add repositories to GitHub App installations without assess to the GitHub administration area (which most developers don't have).

Reviewer: ```suggestion
*Why?* Unfortunately, you can't add repositories to GitHub App installations unless you are an admin of the organisation.
```
